daemon/container.go

```diff
@@ -836,6 +858,8 @@ func (c *Container) AttachWithLogs(stdin io.ReadCloser, stdout, stderr io.Writer
 			logrus.Errorf("Reading logs not implemented for driver %s", c.LogDriverType())
 		} else {
 			dec := json.NewDecoder(cLog)
+			c.LogEvent("attach")
```

Comment:
This block isn't returning from function. So, I think it's enough to Log just before `if stream`. Because there showing all logs is finished and we're about to start operation with indefinite length, if `stream` is `false` then logging this event will be ok too as if we finished to stream immediately.